libs/oidc/client/src/lib/guards/roles.guard.ts

```diff
@@ -1,5 +1,6 @@
-import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { CanActivate, ExecutionContext, Injectable, UnauthorizedException } from '@nestjs/common';
 
+import { AuthUtils } from '../../../../common/src/lib/utils/auth/auth.util';
```

Comment:
Cannot use relative lib imports.

This should show up as a warning in VSC.

Rule: `nx-enforce-module-boundaries`

Reply:
We need to go over how to get my linter working properly